build.gradle.kts

```diff
@@ -369,9 +380,13 @@ tasks {
   runIde {
     dependsOn(project.tasks.getByPath("buildCody"))
     jvmArgs("-Djdk.module.illegalAccess.silent=true")
+    // TODO: Figure out why systemProperties(...) doesn't work
```

Comment:
It's because one was `cody-agent` and the other is `cody.agent`. Fix incoming...

Reply:
Wow, nice catch (and thanks Cody for suggesting the wrong key 😢 )